### ISSUE TITLE
refactor: merge `*RequestDto` with `*UpdateDto` 

### ISSUE THREAD

**Issue by ndr-brt** [eclipse-edc/Connector#2870]
# Feature Request

currently, for the `Asset`, `ContractDefinition` and `PolicyDefinition` endpoints we have two different objects representing the request body for creation and update.
Specifically, the `UpdateDto` miss the `id` property, but in fact it is something that could (and maybe, should) specified in the request body (other than in the request path).

Giving this, we could remove the `WrapperDto` classes and related transformers.

NOTE: this could be done only after the current deprecated is removed, otherwise this will be a breaking change.

## Which Areas Would Be Affected?
management api

## Why Is the Feature Desired?
refactor

## Solution Proposal
_If possible, provide a (brief!) solution proposal._

**Comment by ndr-brt** [comment]
superseded by #3267
